integration_tests/test_fee_token.py

```diff
@@ -113,34 +109,27 @@ def buy_token_from_v2():
         .build(codec.get_default_deadline())
     )
     trx_hash = send_transaction(v2_in_amount, encoded_input)
-
     receipt = w3.eth.wait_for_transaction_receipt(trx_hash)
     assert receipt["status"] == 1  # trx success
-    assert w3.eth.get_balance(account.address) < init_amount - v2_in_amount
+    assert w3.eth.get_balance(account.address) < init_amount - v2_in_amount  # ADD THIS
     assert token_contract.functions.balanceOf(account.address).call() > 0
-    print("Balance after buy:", token_contract.functions.balanceOf(account.address).call())
-
-    print(" => BUY TOKEN: OK")
+    print("Balance after buy:", token_contract.functions.balanceOf(account.address).call())  # CHANGE THIS
 
 
 def sell_token_from_v2():
-    # Selling token with fee
     v2_path = [token_address, weth_address]
     v2_in_amount = token_contract.functions.balanceOf(account.address).call()
-    v2_out_amount = 0  # with slippage
-
+    v2_out_amount = 0
     permit_data, signable_message = codec.create_permit2_signable_message(
         token_address,
-        # amount_in,  # max/infinite = 2**160 - 1
         2 ** 160 - 1,
-        codec.get_default_expiration(40 * 24 * 3600),  # 30 days
-        0,  # Permit2 nonce
+        codec.get_default_expiration(40 * 24 * 3600),
```

Comment:
Ok, here the comment was wrong so fixing it or removing it since it adds little to the understanding is fine.